src/index.html

```diff
@@ -1,12 +1,12 @@
 <!DOCTYPE HTML>
-<html leng="en">
+<html lang="en">
   <head>
-    <meta charset="utf-8">
+    <meta charset="utf-8"> 
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <link 
-      rel="stylesheet" 
-      href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" 
-      crossorigin="anonymous">
+    <link rel="stylesheet" 
```

Comment:
One attribute per line. Take `rel` to the next line

Reply:
I have moved it to a new line.